### ISSUE TITLE
Persist data in DB so you don't lose games when something happens to the bot

### ISSUE THREAD

**Issue by leviatas** [d0tcc/Secret-Blue#20]
I think the title makes clear how useful this could be.

**Comment by d0tcc** [comment]
Yes, it's saved to a textfile now, but a DB would be way better!

**Comment by leviatas** [comment]
It's saved to a text file? Because I wasn't able to continue a game after an update. Both on the original bot and the test bot.

**Comment by d0tcc** [comment]
The status of a game is never saved and I don't plan on implementing this because I don't want to persist every little detail of the game in the DB. At the moment only the stats are saved in a file but for the future (language support) we need to persist group settings as well.

**Comment by leviatas** [comment]
I knew stats were saved as well as logs. 

I'm looking a way to persist a game in its current state so if something happens to the bot (updates/errors/etc) you can continue the game. I plan on saving the game object on a json and allow continue when using the start command If the game isn't in the game controller. 

The stats can be saved as well.
